### PR TITLE
Use similar names between Scala and Java TOCs

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: documentation/manual/working/javaGuide/main/sql/index.toc [hunk 1]
```diff
@@ -1,3 +1,2 @@
 JavaDatabase:Configuring and using JDBC
-JavaJPA:Integrating with JPA
-JavaEbean:Using Ebean ORM
```
Comment (PromanSEW) [inline documentation/manual/working/javaGuide/main/sql/index.toc]: Why delete Ebean?

Comment (marcospereira) [reply documentation/manual/working/javaGuide/main/sql/index.toc]: https://github.com/playframework/play-ebean#support

The idea is to have its documentation published independently as [we have for Anorm now](https://playframework.github.io/anorm/).

Comment (renatocaval) [reply documentation/manual/working/javaGuide/main/sql/index.toc]: @marcospereira, this removing play-ebean from the TOC, but don't we also need to remove it from the generated docs?

Comment (marcospereira) [reply documentation/manual/working/javaGuide/main/sql/index.toc]: @renatocaval this was done here already: https://github.com/playframework/omnidoc/pull/47

**Pull request merged**